yb-voyager/src/query/queryissue/issues_dml.go

```diff
@@ -73,28 +74,48 @@ var regexFunctionsIssue = issue.Issue{
 	Suggestion:  "",
 	GH:          "https://github.com/yugabyte/yugabyte-db/issues/25575",
 	DocsLink:    "https://docs.yugabyte.com/preview/yugabyte-voyager/known-issues/postgresql/#postgresql-12-and-later-features",
+	MinimumVersionsFixedIn: map[string]*ybversion.YBVersion{
+		ybversion.SERIES_2_25: ybversion.V2_25_0_0,
+	},
 }
 
 func NewRegexFunctionsIssue(objectType string, objectName string, sqlStatement string) QueryIssue {
 	return newQueryIssue(regexFunctionsIssue, objectType, objectName, sqlStatement, map[string]interface{}{})
 }
 
-var aggregateFunctionIssue = issue.Issue{
-	Type:        AGGREGATE_FUNCTION,
-	Name:        AGGREGATION_FUNCTIONS_NAME,
+var anyValueAggregateFunction = issue.Issue{
+	Type:        ANY_VALUE_AGGREGATE_FUNCTION,
+	Name:        ANY_VALUE_AGGREGATE_FUNCTION_NAME,
 	Impact:      constants.IMPACT_LEVEL_2,
-	Description: AGGREGATE_FUNCTION_ISSUE_DESCRIPTION,
+	Description: ANY_VALUE_AGGREGATE_FUNCTION_ISSUE_DESCRIPTION,
 	Suggestion:  "",
 	GH:          "https://github.com/yugabyte/yugabyte-db/issues/25575",
 	DocsLink:    "https://docs.yugabyte.com/preview/yugabyte-voyager/known-issues/postgresql/#postgresql-12-and-later-features",
 }
 
-func NewAggregationFunctionIssue(objectType string, objectName string, sqlStatement string, funcNames []string) QueryIssue {
+func NewAnyValueAggregateFunctionIssue(objectType string, objectName string, sqlStatement string) QueryIssue {
+	return newQueryIssue(anyValueAggregateFunction, objectType, objectName, sqlStatement, map[string]interface{}{})
+}
+
+var rangeAggregateFunctionIssue = issue.Issue{
+	Type:        RANGE_AGGREGATE_FUNCTION,
+	Name:        RANGE_AGGREGATE_FUNCTION_NAME,
+	Impact:      constants.IMPACT_LEVEL_2,
+	Description: RANGE_AGGREGATE_FUNCTION_ISSUE_DESCRIPTION,
+	Suggestion:  "",
+	GH:          "https://github.com/yugabyte/yugabyte-db/issues/25575",
```

Comment:
we should create separate GH as well then

Reply:
we don't have separate GH for PG15-17 issues as we had decided to have a single GH and docs link for these features